src/mds/MDSRank.cc

```diff
@@ -1186,24 +1187,36 @@ bool MDSRank::_dispatch(const cref_t<Message> &m, bool new_msg)
 }
 
 #ifdef WITH_CEPHFS_NOTIFICATION
+
+void MDSRank::send_notification_info_to_peers(const ref_t<Message> &m) {
```

Comment:
This looks like a general purpose method if renamed to e.g. send_to_peers().
So can be rather defined/declared unconditionally.